pkg/scheduler/plugins/elasticquota/plugin_helper.go

```diff
@@ -59,6 +61,17 @@ var GetQuotaName = func(pod *v1.Pod, quotaLister schedulinglisterv1alpha1.Elasti
 	} else if !errors.IsNotFound(err) {
 		klog.Errorf("Failed to Get ElasticQuota %s, err: %v", pod.Namespace, err)
 	}
+
```

Comment:
We should change the variable `GetQuotaName` to a function in this PR.

Reply:
fixed